eng/ci/templates/official/jobs/setup-benchmark-agents.yml

```diff
@@ -1,29 +1,51 @@
 parameters:
 - name: functionAppName
   type: string
-- name: agentName
+- name: os
```

Comment:
You can add an `values` section to restrict to linux/windows.

Reply:
Good idea. Added values in both places.